ArmPkg/Drivers/CpuPei/CpuPei.c

```diff
@@ -138,7 +138,8 @@ InitializeCpuPeim (
   ArmEnableBranchPrediction ();
 
   // Publish the CPU memory and io spaces sizes
-  BuildCpuHob (ArmGetPhysicalAddressBits (), PcdGet8 (PcdPrePiCpuIoSize));
+  BuildCpuHob ((UINT8)ArmGetPhysicalAddressBits (), PcdGet8 (PcdPrePiCpuIoSize)); // MU_CHANGE Add typecast

```

Comment:
I am sure this change works.

But would it be more appropriate to update the function prototype to return UINT8 for this function (ArmGetPhysicalAddressBits)? Or do we really see some potential to have a 256+bit CPU in the foreseeable future?

Reply:
Yeah, I was thinking about that same thing. I'd support that approach.